### ISSUE TITLE
Screen reader does not read BYOC model selection

### ISSUE THREAD

**Issue by zuhairmahd** [microsoft/vscode#293702]
Type: <b>Bug</b>

1. Add a model in the Configure Chat Models box.
2. For good measure, restart VSCode (it happens when you do not restart, but as I said, for good measure).
3. Open the chat window and try selecting your added model:
- expected behavior: the model name is spoken followed by "Checked" or "Unchecked" depending on whether it has been selected or not.
- Experienced behavior: the screen reader says "Action"
- If you added more than one model, the screen reader says nothing as you arrow up and down the list of new models instead of speaking them.

VS Code version: Code - Insiders 1.110.0-insider (c308cc9f87e1427a73c5e32c81a1cfe9b1b203d1, 2026-02-06T17:10:19.395Z)
OS version: Windows_NT arm64 10.0.26220
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Snapdragon(R) X 12-core X1E80100 @ 3.40 GHz (12 x 2976)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x4d4f4351, DEVICE=0x36334330 [Qualcomm(R) Adreno(TM) X1-85 GPU], DRIVER_VENDOR=Qualcomm, DRIVER_VERSION=31.0.112.0 *ACTIVE*<br>GPU1: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.6690<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: unavailable_software|
|Load (avg)|undefined|
|Memory (System)|15.61GB (4.44GB free)|
|Process Argv|. --crash-reporter-id ed9bec34-59fd-44a4-9826-062926468e6c|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (52)</summary>

Extension|Author (truncated)|Version
---|---|---
claude-code|Ant|2.1.37
azps-tools|azp|0.3.0
azurite|Azu|3.35.0
codespaces|Git|1.18.5
copilot-chat|Git|0.38.2026020704
remotehub|Git|0.65.2026012801
vscode-github-actions|git|0.31.0
vscode-pull-request-github|Git|0.129.2026020604
gemini-cli-vscode-ide-companion|goo|0.20.0
rest-client|hum|0.25.1
ilspy-vscode|ics|0.23.1
rainbow-csv|mec|3.24.1
azure-pipelines|ms-|1.261.1
microsoft-testing|ms-|0.2.0
azure-dev|ms-|0.10.0
vscode-apimanagement|ms-|1.3.0
vscode-azure-github-copilot|ms-|1.0.153
vscode-azure-mcp-server|ms-|2.0.17
vscode-azureappservice|ms-|0.26.4
vscode-azurecontainerapps|ms-|0.10.0
vscode-azurefunctions|ms-|1.20.3
vscode-azureresourcegroups|ms-|0.11.8
vscode-azurestaticwebapps|ms-|0.13.2
vscode-azurestorage|ms-|0.17.1
vscode-azureterraform|ms-|0.8.0
vscode-azurevirtualmachines|ms-|0.6.10
vscode-containers|ms-|2.4.1
vscode-cosmosdb|ms-|0.31.2
vscode-docker|ms-|2.0.0
vscode-logicapps|ms-|1.2.9
csdevkit|ms-|2.11.4
csharp|ms-|2.121.32
vscode-dotnet-runtime|ms-|3.0.0
remote-containers|ms-|0.444.0
remote-ssh|ms-|0.123.2026012215
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.104.3
vscode-remote-extensionpack|ms-|0.26.0
azure-account|ms-|0.13.0
azure-repos|ms-|0.41.2026012801
azurecli|ms-|0.6.0
powershell|ms-|2025.4.0
remote-explorer|ms-|0.6.2025081809
remote-repositories|ms-|0.43.2026012801
remote-server|ms-|1.6.2026020509
vscode-node-azure-pack|ms-|1.8.0
windows-ai-studio|ms-|0.30.0
chatgpt|ope|0.5.72
pester-test|psp|2023.7.7
vscode-xml|red|0.29.2025112508
vscode-yaml|red|1.20.2026020708
vscode-ai-foundry|Tea|0.14.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
pythonvspyt551:31249597
nativeloc1:31118317
dwcopilot:31158714
dwoutputs:31242946
copilot_t_ci:31333650
g012b348:31231168
pythonrdcb7:31268811
pythonpcpt1cf:31399617
6518g693:31302842
6abeh943:31336334
envsactivate1:31349248
editstats-enabled:31346256
cloudbuttont:31366566
3efgi100_wstrepl:31403338
bj468946:31457056
use-responses-api:31390341
je187915:31401257
cp_cls_t_966_ss:31454198
chatvisible:31417128
inlinechat_v2_hd992725:31445440
4je02754:31455664
ge8j1254_inline_auto_hint_haiku:31426887
a5gib710:31434435
2984c849_gpt52_codex:31457542
7ig2g208:31429686
722j7530_gpt_5_2_codex:31458091
406hc587_ask_agent:31428392
preserve_tokens:31444547
cp_cls_c_1081:31454833
copilot-nes-oct-trt:31432609
ia-use-proxy-models-svc:31446143
nes-slash-models-on:31441832
a43f0575b:31442821
e9c30283:31453065
c0ci8992_validated:31449878
h_5b0j5247:31457745
nes-trig-10:31449242
idci7584:31454084
edit_mode_hidden:31453871
nes-ftch-new:31458522
nes_chat_context_disabled:31451402
showstats:31452402
e3e4d672:31454087
864ei723_large_tool_results_to_disk:31455802
showingstats:31457201
pp-44cc9697:31455969
nes-fast-ctrl:31454625
i54ji102:31455230
55364912:31455227
hg17d649:31455236
nes-extended-on:31455475
nes-sp-on:31458511

```

</details>
